editoast/migrations/2023-09-06-161017_initial/up.sql

```diff
@@ -0,0 +1,2406 @@
+--
+-- PostgreSQL database dump
+--
```

Comment:
It will be painful to prettify the 2000 lines of SQL. I can rename constraints that should be renamed but I don't see the need to spend more effort in it.

Reply:
As discussed by message, the idea is to look for a prettier pg schema dumper, and give up if we don't

Reply:
I have generated it with dbeaver, it works.